### PR TITLE
Add com.jeffser.Alpaca

### PR DESCRIPTION
**Pull request by Jeffser**
### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

## Description
My app is a client for [Ollama](https://ollama.com/), the user can chat with multiple models in the same chat and they can also remove and pull models. I'm still developing the app but I think it is ready to be published.

### Note
I'm the owner of jeffser.com and the only developer of this application, the only dependency I use is python-requests, if there is a better way of including that on the manifest please tell me.

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.jeffser.Alpaca

File: com.jeffser.Alpaca.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"requests\" --no-build-isolation"
+	    ],
+	    "sources": [
+		{
+		    "type": "file",
+		    "url": "https://files.pythonhosted.org/packages/ba/06/a07f096c664aeb9f01624f858c3add0a4e913d6c96257acb4fce61e7de14/certifi-2024.2.2-py3-none-any.whl",
+		    "sha256": "dc383c07b76109f368f6106eee2b593b04a011ea4d55f652c6ca24a754d1cdd1"
+		},
```
Comment (bbhtt) [inline com.jeffser.Alpaca.json:37]: Indentation is wrong here and below

**Comment by bbhtt** [comment]
> My app is a client for [Ollama](https://ollama.com/), the user can chat with multiple models in the same chat and they can also remove and pull models. I'm still developing the app but I think it is ready to be published.

DOes this require installing ollama seperately and pulling the models from cli? If so please mention the instructions for that in the first line of `description` tag of your metainfo file

Or preferably you have some sort of first run popup with instructions?

**Comment by bbhtt** [comment]
Yes please. I imagine most people will be running a local instance.

Also preferences window doesn't seem to open.


https://github.com/flathub/flathub/assets/62639087/5afbc820-b200-4661-afdf-1aee198f8b47

**Comment by Jeffser** [comment]
Oh yeah you are right I totally forgot about the preferences menu, I'll have it done in a couple of hours.

**Comment by Jeffser** [comment]
Hi I just finished adding a welcome / explanation dialog.
I also changed the preferences button for two different buttons:
- Clear conversation
- Reconnect (shows the setup dialog)

I also made it so that the conversation and server url are persistent after the user closes the app

I changed the manifest so that it points to the new tag

thanks

**Comment by Jeffser** [comment]
[Screencast from 2024-05-13 14-15-11.webm](https://github.com/flathub/flathub/assets/69224322/eba3b34c-4ce9-4e46-97f7-e99fa8634001)

**Comment by bbhtt** [comment]
bot, build com.jeffser.Alpaca

**Comment by Jeffser** [comment]
Hi thanks for the feedback, I just updated the app, it should be good now

**Comment by bbhtt** [comment]
bot, build com.jeffser.Alpaca

**Pull request closed without merging**